docs/manual_installation.md

```diff
@@ -316,12 +406,13 @@ sudo systemctl disable networking.service
 sudo systemctl disable dphys-swapfile.service
```

Comment:
this might no longer be needed, service "is masked, ignored" after previously removing it in the "Further OS Modifications" section.